client/web/src/insights/components/visibility-picker/VisibilityPicker.tsx

```diff
@@ -67,12 +67,17 @@ export const VisibilityPicker: React.FunctionComponent<VisibilityPickerProps> =
         }
     }
 
+    const possibleOrg = organizations.find(org => org.id === value)
+
+    const descriptionText = !possibleOrg?.displayName
```

Comment:
This will currently show the first message if there is an org, but it doesn't have a `displayName` (which is nullable), which would be wrong.
```suggestion
    const descriptionText = !possibleOrg
```